include/draw_buffer.h

```diff
@@ -0,0 +1,16 @@
+#ifndef GUARD_DRAW_BUFFER_H
+#define GUARD_DRAW_BUFFER_H
+
+#include "global.h"
+
+extern struct OamData gUnk_030031C0[];
+extern u8 gUnk_03002450[];
+extern u8 gUnk_030024F0;
+extern u8 gUnk_030035F0[];
+extern u8 gUnk_03003A00;
+extern u8 gUnk_03006080[];
+extern struct OamData gOamBuffer[];
```

Comment:
For symbols already in other header (e.g. `gUnk_030031C0`, `gOamData`), we'd better not declare them again here
Otherwise put them in include/data.h because we have no idea of the RAM map yet (especially where we should divide `.bss`/`COMMON`)